### ISSUE TITLE
Error when autoloading .tim file: Key not present in Dictionary: chunk_first

### ISSUE THREAD

**Issue by sjl** [tpope/timl#12]
So after #11 I moved my `.tim` code into `autoload/` and added a `.vim` stub in `plugin/`, and now I get the following error when I run the stubbed command (which calls the TimL code):

```
Error detected while processing function <SNR>72_autoload..timl#loader#source..timl#compiler#build..<SNR>130_emit..timl#call..326..timl#type#apply..timl#lazy_seq#seq..<SNR>132_val..t
iml#call..468..<SNR>107_predicate..timl#coll#chunked_seqp:
line    1:
E716: Key not present in Dictionary: chunk_first)
E116: Invalid arguments for function timl#type#canp(a:coll, g:timl#core.chunk_first)
E15: Invalid expression: timl#type#canp(a:coll, g:timl#core.chunk_first)
Error detected while processing function <SNR>72_autoload:
line   10:
E170: Missing :endfor
Press ENTER or type command to continue
```

I probably fucked something up, but it'd be nice to have a friendlier error.
